client/src/components/App.jsx

```diff
@@ -1,39 +1,87 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import TitleBar from './TitleBar.jsx';
 import ImagesList from './ImagesList.jsx';
+import axios from 'axios';
 
-var mockData = {
-  room_name: 'Modern Lounge Ultra clean',
-  location_city: 'Santa Barbara',
-  location_country: 'United States',
-  average_review_point: '4.92',
-  number_of_reviews: 843,
-  is_superhost: true,
+var initialState = {
+  room_name: '',
+  location_city: '',
+  location_country: '',
+  average_review_point: '',
+  number_of_reviews: '',
+  is_superhost: '',
   images: [
-    'https://rooms-images-128.s3-us-west-1.amazonaws.com/image1.jpg',
-    'https://rooms-images-128.s3-us-west-1.amazonaws.com/image2.jpg',
-    'https://rooms-images-128.s3-us-west-1.amazonaws.com/image3.jpg',
-    'https://rooms-images-128.s3-us-west-1.amazonaws.com/image4.jpg',
-    'https://rooms-images-128.s3-us-west-1.amazonaws.com/image5.jpg'
-  ]
+    '','', '', '', '']
 };
 
 class App extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
-      data: mockData,
+      error: null,
+      isLoaded: true,
+      data: initialState,
     }
+    this.getRoomData = this.getRoomData.bind(this);
+  }
+
+  getRoomData (id) {
+    axios.get(`/api/rooms/${id}`)
+      // .then(response => response.json())
```

Comment:
You could delete the comment if you don't use it anymore.

Reply:
Thanks. Nice catch. Yep, updated with a new commit.